src/core/server.js

```diff
@@ -800,7 +810,10 @@ Server.prototype._sendMessage = function(message) {
     if (this._remote.trace) {
       log.info(this.getServerID(), 'request:', message);
     }
-    this._ws.send(JSON.stringify(message));
+    this._ws.send(JSON.stringify(message), () => {
+      // sometimes gitves 'not opened'
```

Comment:
I think what happens - is that `ws` internal state is updated (after connection dropped), but `Server's` isn't, so `Server` tries to send message, but `ws` gives error. But then `ws` emit `onerror` (or `onclose`, don't remeber) event, and `Server` standard reconnection starts.

Message gets lost. Do you think we should re-send it on reconnect?